src/frontend/packages/lib_video/src/api/setLiveSessionDisplayName/index.ts

```diff
@@ -15,7 +15,7 @@ export const setLiveSessionDisplayName = async (
   const response = await fetch(`${API_ENDPOINT}/livesessions/display_name/`, {
     body: JSON.stringify(body),
     headers: {
-      Authorization: `Bearer ${useJwt.getState().jwt}`,
+      Authorization: `Bearer ${useJwt.getState().jwt ?? ''}`,
```

Comment:
It should maybe just return an error and not call fetch if jwt is empty.

Reply:
IDK, I did not change the implementation and since we are not providing a jwt, the backend should raise an error if that occur (and I we do change this behavior we will have to change a lot of test to provide a jwt :( )